modules/cms-kit/src/Volo.CmsKit.Admin.Web/Pages/CmsKit/Comments/Details.cshtml

```diff
@@ -93,6 +94,17 @@
                             <abp-column size-lg="_4" size-md="_12">
                                 <abp-input asp-for="@Model.Author" label="@L["Username"].Value" type="text"/>
                             </abp-column>
+                            <abp-column size-lg="_2" size-md="_12">
+                                <div class="mb-3" id="IsApprovedSelectInput">
+                                    <label class="mb-1" for="IsApprovedSelect">@L["CommentFilter:ApproveState"].Value</label>
+                                    <select id="IsApprovedSelect" asp-for="@Model.CommentApproveState" class="form-control">
+                                        <option value="@CommentApproveState.All">@L["CommentFilter:All"].Value</option>
+                                        <option value="@CommentApproveState.Approved">@L["CommentFilter:Approved"].Value</option>
+                                        <option value="@CommentApproveState.Disapproved">@L["CommentFilter:Disapproved"].Value</option>
+                                        <option value="@CommentApproveState.Waiting">@L["CommentFilter:Waiting"].Value</option>
```

Comment:
I think instead of listing options like this one by one, you can use `Enum.GetValues` or `Enum.GetNames` method, and dynamically list the options. 

Example:

https://github.com/volosoft/volo/blob/dfcaf6e23805a5b9c3b1e04b44c866dbc45d37cc/abp/saas/src/Volo.Saas.Host.Blazor/Pages/Saas/Host/TenantManagement.razor#L104-L109

If you would not do it like that, then when a new approve state needed to be added, then you should manually update here.